lib/opensusebasetest.pm

```diff
@@ -767,10 +767,13 @@ sub select_serial_terminal {
         } else {
             $console = $root ? 'root-virtio-terminal' : 'virtio-terminal';
         }
-    } elsif (get_var('S390_ZKVM')) {
-        $console = $root ? 'root-console' : 'user-console';
-    } elsif ($backend eq 'svirt') {
-        $console = $root ? 'root-console' : 'user-console';
+    } elsif ($backend eq 'svirt' || get_var('S390_ZKVM')) {
```

Comment:
`S390_ZKVM` is always with svirt backend, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/0acc8501320f854342a7e6dc3c1162ae9af2bf95/lib/susedistribution.pm#L369

Reply:
@mnowaksuse Thx, I'll use only `$backend eq 'svirt'`).